R/tax_unique.R

```diff
@@ -0,0 +1,279 @@
+#' Filter occurrences to unique taxa
+#'
+#' A function to filter a list of taxonomic occurrences to unique taxa of a
```

Comment:
I think we should reduce this section of the description and move some of the information to the details section. I think you can make reference to the details here though.